selfdrive/car/hyundai/fingerprints.py

```diff
@@ -1706,5 +1706,16 @@
     (Ecu.fwdRadar, 0x7d0, None): [
       b'\xf1\x00US4_ RDR -----      1.00 1.00 99110-CG000         ',
     ],
-  },
+  },  
+ CAR.KIA_EV9: {
+    (Ecu.fwdRadar, 0x7d0, None): [
+      b'\xf1\x00MV__ RDR -----      1.00 1.02 99110-DO700         ',
+    ],
+    (Ecu.adas, 0x730, None): [
+      b'\xf1\x00MV  FBL5 1.00 1.02 230602',
+    ],
```

Comment:
```suggestion
```

Reply:
Why suggest removing this? After fingerprinting my car I have this fingerprint but a different value.

Reply:
We don't fingerprint on the adas module because it's not always there for every variant of the platforms we've seen (HDA 1 vs HDA 2). Camera and radar are usually enough to fingerprint